### PR TITLE
Input: decouple from redux forms

### PR DESCRIPTION
**Pull request by driesd**
### Description

This PR decouples our `Input` component from `redux forms`. Therefore we removed the `input` and `meta` props. Also we added an `error` prop which can accept either a string or an element. The value of the `error` prop will be rendered below the input field just like `meta.error` was before this PR.

Solves #247 

### Breaking changes

We removed the `input` and `meta` props to decouple from redux-forms.

### CODE REVIEW

File: components/input/Input.js [hunk 1]
```diff
@@ -17,9 +15,9 @@ import theme from './theme.css';
 
 export default class Input extends PureComponent {
   static getDerivedStateFromProps(nextProps, prevState) {
-    const value = nextProps.input ? nextProps.input.value : nextProps.value;
-    if (value !== undefined) {
-      const newValue = Input.parseValue(Input.getPropsValue(nextProps), nextProps);
+    if (nextProps.value !== undefined) {
```
Comment (JorenSaey) [inline components/input/Input.js]: Is it on purpose that null is a valid value and undefined is not?

Comment (driesd) [reply components/input/Input.js]: Yes it's on purpose. It could be that the parent container wants to set the value to null.

Comment (timdegroote) [reply components/input/Input.js]: what this check is doing is checking wether a value has been supplied by the parent component (i.e. if this is not the case the value of `value` will be `undefined`).

If a parent decides to for whatever reason pass `null` then it means the parent has passed a value and this component shouldn't care about it anymore.

If a value is indeed `undefined` though, the Input component will AFAIK keep track of it in its state etc.

Comment (JorenSaey) [reply components/input/Input.js]: Thanks for the explanation :)

**Pull request merged**